apps/mull-ui/src/app/components/subnavigation-bar/subnavigation-bar.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import { createMemoryHistory } from 'history';
+
+import SubNavigationBar from './subnavigation-bar';
+import { Router } from 'react-router-dom';
+import { ROUTES } from '../../../constants';
+
+describe('SubNavigationBar', () => {
+  it('should render successfully', () => {
+    const history = createMemoryHistory();
+
+    const { baseElement } = render(
+      <Router history={history}>
+        <SubNavigationBar />
+      </Router>
+    );
+
+    expect(baseElement).toBeTruthy();
+  });
+
+  it('should have the correct button active based on the url', () => {
+    const testIds = {
+      [ROUTES.DISCOVER]: 'subnavigation-discover-button',
+      [`${ROUTES.UPCOMING}`]: 'subnavigation-upcoming-button',
+      [`${ROUTES.MYEVENTS}`]: 'subnavigation-myEvents-button',
```

Comment:
Same comment as in app.tsx, is there a reason we can't put:
```suggestion
      [ROUTES.UPCOMING]: 'subnavigation-upcoming-button',
      [ROUTES.MYEVENTS]: 'subnavigation-myEvents-button',
```
I tried it and the tests still seem to work.

Reply:
It's a remnant from my first iteration, I was doing the routes a bit different on this component, should be what you proposed now.